test_skype_send_message.py

```diff
@@ -1,37 +1,41 @@
 import unittest
 import skype_send_message
+import keyring
+
+
 class skype_send_message_test(unittest.TestCase):
```

Comment:
@idumenskyi Python uses CamelCase as for the class names and underscore_names as for the variables and methods/function names. Lets use `SkypeSendTextMessageTest`